pkg/cli/roundtripper.go

```diff
@@ -24,15 +24,15 @@ import (
 	"strings"
 )
 
-type retryingRoundTripper struct {
+type RetryingRoundTripper struct {
```

Comment:
The better place for this would probably be `pkg/kube`, as this contains other Kubernetes client related things.

Reply:
Sure, I can move it. My initial approach was to make the bare minimum changes....

Reply:
Updated, thanks for having a look.